src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -29,6 +29,7 @@ public partial class ScriptEditorWindow : ModelessChildWindow
         public PythonNode nodeModel { get; set; }
         private bool nodeWasModified = false;
         private string originalScript;
+        internal NodeView nodeView;
```

Comment:
I don't see this field being used. Can we remove it?

Reply:
Sure removed